server.py

```diff
@@ -12,6 +13,7 @@
 from modules.constants import HttpResponse, http_code_to_enum
 from modules.metrics import MetricsHandler
 
+ROWS_PER_PAGE = 10
```

Comment:
this should go in `modules/sqlite_helpers.py` instead

Reply:
done